intern/remote/boss/remote.py

```diff
@@ -850,6 +858,22 @@ def delete_metadata(self, resource, keys):
         self.metadata_service.set_auth(self._token_metadata)
         self.metadata_service.delete(resource, keys)
 
+    def parse_bossURI(self, uri): # type: (str) -> Resource
+        """
+        Parse a bossDB URI and handle malform errors.
+
+        Arguments:
+            uri (str): URI of the form bossdb://<collection>/<experiment>/<channel>
+
+        Returns:
+            Resource
+
+        """
+        t = uri.split("://")[1].split("/")
+        if len(t) is 3:
+            return self.get_channel(t[2], t[0], t[1])
+        raise ValueError(f"Cannot parse URI {uri}.")
```

Comment:
Sorry, missed this the first two times. One more `f` formatted string to remove.